Core/src/org/sleuthkit/autopsy/datamodel/OsAccounts.java

```diff
@@ -111,34 +111,46 @@ public String getItemType() {
         private final PropertyChangeListener listener = new PropertyChangeListener() {
             @Override
             public void propertyChange(PropertyChangeEvent evt) {
-                refresh(true);
+                String eventType = evt.getPropertyName();
+                if(eventType.equals(Case.Events.OS_ACCOUNT_ADDED.toString())) {
+                     refresh(true);
+                } else if (eventType.equals(Case.Events.CURRENT_CASE.toString())) {
+                    // case was closed. Remove listeners so that we don't get called with a stale case handle
+                    if (evt.getNewValue() == null) {
+                        removeNotify();
+                        skCase = null;
+                    }
+                }
             }
         };
         
         @Override
         protected void addNotify() {
             Case.addEventTypeSubscriber(Collections.singleton(Case.Events.OS_ACCOUNT_ADDED), listener);
+            Case.addEventTypeSubscriber(EnumSet.of(Case.Events.CURRENT_CASE), listener);
```

Comment:
I needed to import EnumSet

Reply:
Line 25 had the EnumSet import.

Reply:
I pulled in develop just to make sure it would build with what I have for sleuthkit and that removed the import.  I guess some other work has been happening in this file that is overwriting.  Would you please pull in latest from develop?

Reply:
Seems like something happens when you merged in develop.  It should be all set now.

Reply:
Great.  Yes, that's working for me now.  Thanks!